sbibm/algorithms/abcpy/abcpy_utils.py

```diff
@@ -0,0 +1,124 @@
+from typing import Callable
```

Comment:
Could you run `isort` and `black` on the files of the PR? That will take care of sorting imports and keeping the code formatting consistent. Some short info about these two is here: https://github.com/sbi-benchmark/sbibm/blob/main/CONTRIBUTING.md#code-style

Reply:
Done